js/packages/openinference-vercel/src/utils.ts

```diff
@@ -368,12 +368,51 @@ const getOpenInferenceAttributes = (attributes: Attributes): Attributes => {
           };
         case AISemanticConventions.TOKEN_COUNT_COMPLETION:
         case AISemanticConventions.TOKEN_COUNT_PROMPT:
+          // Do not capture token counts for chain spans as they are captured by the LLM spans below
+          // Capturing on chains as well results in double token counts
+          if (spanKind === OpenInferenceSpanKind.CHAIN) {
```

Comment:
could flip this conditional and only capture for llm span kinds, currently this is the same either way since there are only llms and chains that have these attributes but might be more future proof if we don't expect any other span kind to care about token counts.

Reply:
I think you should flip this.